### DISCUSSION TITLE
Daily Standup

### DISCUSSION THREAD

**Discussion by mnichols08** [chingu-voyages/v43-tier1-team-09#82] [Show and tell]
This is the official thread for daily standups.

Please participate daily and let us know
1. What did you work on today?
2. What are you struggling with?
3. Who deserves a shout out?
4. Something else about your life non-project related to share.

**Comment by mnichols08** [comment]
Published 9:07PM on 3/21 to [Discord_Chingu>v43-tier1-01-#tier1-team-09#Daily Status](https://discordapp.com/channels/330284646283608064/1087796931750797402/1087905404622749737)
1.  Did some pair programming with @PaulaR-05, finished up several tasks on the project including some I forgot about. Great meeting today thanks so much for helping organize it beforehand @Emimint . We did miss you @emilio12345 hope all is well and that you are able to find some free time to stay on top of our workflow. 
2.  My microphone is causing us issues with communication so I am going to try and use my webcam mic instead.
3. Shout out to @Emimint for putting together a document to help the meeting go smooth and to @PaulaR-05 for all of the support behind the scenes
4. Decided to upgrade my hard drive yesterday. Went from this: 500mb free on my C Drive to 900gb free XD
![image](https://user-images.githubusercontent.com/33106391/226951740-f087fda0-a92f-40ff-929e-a05a3f9906f4.png)
![image](https://user-images.githubusercontent.com/33106391/226951793-5d095197-406d-4d5a-9f7b-395d17d447f6.png)

**Comment by PaulaR-05** [comment]
1. Did some pair programming with @mnichols08, I could understand/learn how to use Github on terminal and basic commands to work with. The meeting was great, I can see that we are all improving our soft skills and building positive relationships with each other. Thanks so much for helping organize it beforehand @Emimint, it was much simpler using that document you suggested. Also, thanks a lot @mnichols08, I can see that you are a great teammate, always helping us to succeed on this project.
2. My PC was slow, so I did some changes but anyway I will try to make sure everything is working well.
3. Shout out to @everyone, 2 more items on the sprint, and its only Wednesday. YAY!
4. I'm thinking about adopting a dog, I had one for 18 years but she passed away in December, last year. I miss her so much :'(

**Reply by Emimint** [reply]
Awww sorry about your puppy :(  ! I undestand how you feel. So many doggies are looking for a home, go make one happy :) !

**Comment by Emimint** [comment]
1) **Today I**: worked on creating a mobile friendly CSS file for the landing page. 

2) **I am struggling with**: CSS in general haha, 😅! 

3) **Special Thanks to**: Gabriela for accepting to help me tomorrow with making this work 🙏🏾 . 

4) **Something else random about your day**: spring is finally here 🌼💐🌻 and I can't wait to go back to do some running outside 🏃🏾‍♀️!

**Reply by mnichols08** [reply]
http://www.flexboxdefense.com/
https://flexboxfroggy.com/ 
https://mastery.games/flexboxzombies/
https://codingfantasy.com/games/flexboxadventure

Some examples of fun web apps that help you visualize how flex works and subconsciously trick your brain into just getting it hope one of these helps!

**Reply by PaulaR-05** [reply]
Thanks a lot for these examples 🙌

**Reply by Emimint** [reply]
@mnichols08 thanks a lot, I will definitively have a look 🙏🏾!

**Comment by mnichols08** [comment]
1. Today I set up this discussions feature for some things like this. I think if we all commit to communicating their stand up here it will pay off. I do wish I knew about it sooner, in future voyages I plan to use them a lot more and early on.

2. My biggest struggle at the moment regarding this project is seeing a day go by like today where I was available to help someone but didn’t.  If there is ever a time where my time aligns with your time and you have any questions no matter how mundane please let me help you understand =) I enjoy teaching and I grow/learn from it is a win-win trust me!

3. Special Thanks to everyone for supporting each other throughout the project.

4. I had a pretty unique pair programming experience tonight. He proposed it super early on - something neither of us knew how to do. This is what we came up with together https://mnichols08.github.io/chingu-pairprogramming/session-03/dist/index.html. It turned out pretty neat.

**Reply by PaulaR-05** [reply]
Great project, BTW! Congrats 🎉🎊

**Reply by Emimint** [reply]
@mnichols08 I did some pair programming with @PaulaR-05 and you are next on my list! (Well, when our schedule will match 😅 !)

**Reply by PaulaR-05** [reply]
Lucky girl, Em! He is the best teacher ever.

**Reply by mnichols08** [reply]
> @mnichols08 I did some pair programming with @PaulaR-05 and you are next on my list! (Well, when our schedule will match 😅 !)

We will figure out a time, it is a shame we really only have once a week that times align - but that's better than some =)

**Comment by PaulaR-05** [comment]
1. **Today:** @Emimint and I are having a pair programming experience. Yesterday, I wrote a semantic HTML for the game input (forgot to tell you guys 🙁 sorry about that).

2. **I am struggling with:** Making time for studying, I feel that I could help more...But at the same time, I'm having a busy week at work. So, I will try to find a balance between them.

3. **Special Thanks to**: @mnichols08 who has been a great teammate, helping everyone in the project. Thanks a lot, Mikey!

4. **Something else random about your day:** I was struggling with the flu, however today I'm feeling better. Looking forward to getting well very quickly.

**Comment by Emimint** [comment]
1) **Today I**:  finished writing the CSS for the mobile friendly version of the website 🥳! (Still need validation tho 🙃)

2)  **I am struggling with**: Getting organized between studying, taking care of me and my home. Well, it's life.

3) **Special Thanks to**: Gabriela and Mikey for being continuous supply of good tips and advice. You guys are amazing 👍🏾!

4) **Something else random about your day**: I am of to see the last John Wick tomorrow. Can't wait 💕!

**Comment by mnichols08** [comment]
1. Today was draining for me. I am car pooling for the time being so after work I wound up tagging along to run some errands with my friend.  It wasn't a big deal but it added about two extra hours to my commute home which is usually ten minutes. I'm pooped, but I did take some time to review a pull request. I hope it is helpful. 

2. I wish I was able to access the Discord App at work but I have to use my phone. I tend to type more eloquent responses than I will scratch with my phone. I also like to help but I end up being that guy at work that is always answering the phones and juggling tasks so please understand anything I may say from my phone should be taken lightly.

3. Special Thanks to: My coworker Aron for picking me up this morning and giving me a ride to/from work (and tomorrow too!)

4. This whole stand up has been about my day outside of the project haha, so well about the project: Remember our focus is MVP. We don't need it to be fancy, just functional. We are two issues away from writing a few JS files to make it all work - so we can then decide which way to go next. Keep at it we got this!

**Reply by PaulaR-05** [reply]
Omg 2 extra hours after work is hard :( But happy that you could find a ride to work/home and get there safely! :)

**Comment by PaulaR-05** [comment]
1. **Today:** Friday usually is a busy day at work [Reduce to tears]. I almost didn't have time even for a coffee which is considered a crime in Gabriela's world.
2. **I am struggling with:**  Making time for studying. I feel I could help more my teammates. So I'm thinking about waking up early and studying before work.
3. **Special Thanks to**: My brother. He brought me chocolate made in Bahia. produced by the family farming system (it brings together people committed to working on family farming and agroecology ) <3
4. **Something else random about your day:** Tomorrow, I might go to a Foster Event. If everything goes well, I will be able to fill out the form for those interested in adopting...so Fingers Crossed🤞

**Reply by mnichols08** [reply]
No coffee no bueno!

Fingers crossed on the event that it leads to a new family member =)

**Reply by PaulaR-05** [reply]
No coffeee = no bueno!
![NatGeoGIF](https://user-images.githubusercontent.com/118011195/227671772-3a33583b-0a1a-40bc-9af2-b149ed94b86f.gif)


Yay... fingers crossed! 

![ByuntearMemeGIF](https://user-images.githubusercontent.com/118011195/227672003-4b45a985-e543-4c80-bad8-cd1bd7e4e9b3.gif)

**Comment by mnichols08** [comment]
1. Today: I carpooled to work but got to take a company vehicle home. So I got home at a decent time and a ride in the morning at my leisure. That is nice =) Tonight I will still do some more coding but today I have been looking over some code written by @Emimint. I provided a bit of feedback about what is happening. I still have not given a review of @Nazile-Tag's code but it looked to be sufficient however we had some pull requests being made into the wrong repository so some small technical errors, but we are still way on track.

2. I understand everybody here have their own schedules and families and lives, but we really would benefit from coding sessions together. Even if is just ten minutes before work, right @PaulaR-05? 

3. Special Thanks to: @Medic1111 for providing guidance and support

4. I guess it's more of a struggle, but I didn't get a green square on github yesterday because the single contribution I made was a code review that I edited today and it counted as todays =(

**Reply by PaulaR-05** [reply]
Yeah! I'll manage to make more time for it next week. Pair programming was one of the best learning experiences I have ever had!
Also, the last time we were together was a great meeting. I wish we could do it more often during the week 🤞

**Reply by Emimint** [reply]
@mnichols08 sorry about that 😓 , but I think in the end it is valuable experience for all of us. And please prioritize fixing your car 🚗🙏🏾!

**Comment by mnichols08** [comment]
1. Today I worked on a forked version of our project to fiddle around with a vanilla JS html renderer function that I made. It was fun to play with but it has some weird side effects like rendering commas between elements that are mapped.
2. I need to change the clutch in my car and it is appearing that I might have an opening on this coming Tuesday, I am just sad cause I want to be here to.
3. Special Thanks to My Co Worker @zed34k for taking me home this evening and giving me a ride tomorrow. Sucks to not have my car but I am getting by
4. It is really windy here, shaking the whole house at times, hope it chills out soon

**Comment by Emimint** [comment]
1) **Today I**:  I had a quick talk with @mnichols08 about my PR and I am really glad we talk and find a solution 🙏🏾!

2)  **I am struggling with**: I actually did pretty good juggling with life this week. So not much struggle.

3) **Special Thanks to**: the only two other girls among my classmates. We are working hard together 🦸‍♀️!

4) **Something else random about your day**: John Wick 4 was good, but not as much as the third one. My opinion: wait for it to go on TV!

**Reply by PaulaR-05** [reply]
Oh really, are there only 3 girls among your classmates? GIRL POWER o/
![VoteRosietheRiveterGIF](https://user-images.githubusercontent.com/118011195/228008781-c2e484f3-ea90-468d-be65-fa168c6a09fb.gif)

**Comment by mnichols08** [comment]
1. I was thrilled to meet briefly with @Emimint early this morning and discuss the project. Today I had a crazy hectic day at work so I came home and took a power nap but after I:
- opened a pull request to close issue #93 as discussed in #79 
- created issues #96, #97, #98, #100, #101, #103 
- Made a [prototype](https://initialize-forked-madlib.netlify.app/) of what our app could look like if we completed those issues
3. I am struggling with not having enough hours in the day, and also hey I ran out of time but I am here typing so you should be too ;) If you are thank you as well. Good night Team
4. Special thanks to @Nazile-Tag, @PaulaR-05, @emilio12345, and @Emimint for participating in this voyage with me.

**Reply by PaulaR-05** [reply]
Happy you could take a power nap, sometimes we have to slow down and rest! ❤️ 
Also, thanks a lot for all this work, Mikey! 

PS: I forgot to type about my day, yesterday. Sorry about that team 🙁

**Comment by PaulaR-05** [comment]
1. **Today** I: I will be working on the tasks I assigned 🤞 I hope I will be able to do everything
2. **I am struggling with**: well, I'm struggling with a JS course that I'm taking. But I believe it is okay since JS is challenging. I might struggle  in the project this week as we are creating functions and coding, so let's be optimistic (always 🤞 )
3. **Special Thanks to:** everyone  for the hard work you have put into the project :) 
4. **Something else random about your day:** at the weekend, I filled out a form for adopting a puppy. Get our hopes up for it ❤️

**Comment by mnichols08** [comment]
1. Today I reviewed a change to the mobile design by @Nazile-Tag and filled out the sprint review. I also went and tried to give a high level overview of what each function should do and how they should work together.
2. I am sleepy, so I will rest soon.
3. Special Thanks to my coworker Ryan for giving me a ride after work to get [Sapphire](https://www.dsmtuners.com/dsm-profiles/sapphire.15180/) from storage
4. Just looking forward to our meeting tomorrow =)

**Comment by PaulaR-05** [comment]
1. **Today** I: I'm still working on the tasks I assigned, plus we have our meeting :)

2. **I am struggling with**: I'm working with one function I'm responsible for. Although, I'll try to come up with a solution before asking for help 🤞 

3. **Special Thanks to:** Mikey, for always sharing his knowledge with us. And, Em for always being friendly and helpful with our struggles.

4. **Something else random about your day:** it is meeting day, looking forward to it.

That's it... Hope you all have a lovely day <3

**Comment by PaulaR-05** [comment]
1. **Today** I: I'm still working on the tasks I assigned, plus I have pair programming with Mikey.

2. **I am struggling with**: Yesterday I was struggling with a task, but ended up working well o/. But I know this week is going to be hard, once is JS week. So eventually,  there will be challenges and we will come up with a solution, hopefully. 🤞 

3. **Special Thanks to:** everyone, enjoyed meeting you guys yesterday!

4. **Something else random about your day:** anything random, sorry :( 

Hope you all have a lovely day 🙌

**Comment by mnichols08** [comment]
1. Today was successful I believe. I am happy we were able to knock out several issues. @PaulaR-05 and I spent many hours solving an issue where we had a resolved promise but no way to use the data....After much deliberation it turns out the first function we wrote was perfectly fine but we had to use an `await` clause in the next function we wrote to actually handle the data being passed in. After doing so we now have a dynamically rendered story selection page XD I also was able to meet with @Emimint briefly about some of the things she has questions about. I want to thank her again for working on the tedious task of data scrubbing! And @emilio12345 we weren't able to actually help but I am still thrilled you were able to fix the issue preventing you from pushing code to the repository. 
2. I am struggling in my forked app with coming up with a dynamic approach to rendering the story page. In my current model I am forced to manually define each variable name to replace and what to replace it with. I can even print `{{variableName}}` dynamically and try to use .replace(thatString, givenName) and I just can't figure it out. So I have a switch statement that checks what story to render instead. It works but I would love to figure out how to dynamically solve it.
3. Special thanks for putting in the effort this week boys and girls!
4. I got my brain workout in for the day that is for sure! Hope all of you have a good night.

**Comment by PaulaR-05** [comment]
1. Today I: I'm feeling tired, yesterday we had a difficult challenge to solve although it was also an amazing learning experience. I'm planning to break all the code again and make sure that I understood how and why the function works. (PS: thanks again Mikey o/)

2. I am struggling with: the JS week + the whole translating thing has been tough, however lots of useful skills to learn 🤞 

3. Special Thanks to: @mnichols08. For always being helpful! Yesterday we spent many hours solving an issue and thanks god it ended up working well haha. Also, I want to thank him again for teaching, working, and for always being an awesome teammate. Thanks a lot, Mikey! (I know that because of the language, teaching can be really challenging hahaha...so, thanks again really!)
  
4. Something else random about your day: I'm gonna make Thai Cashew Chicken stir fry for dinner. https://www.recipetineats.com/thai-cashew-chicken-stir-fry/ . 🧑‍🍳 

Hope you all have a lovely day 🙌

**Comment by emilio12345** [comment]
- Today: I'm planning on actually contributing to the project by tackling at least one issue. I have some documentation from @mnichols08 that I need to review on our git workflow to make sure I'm doing things correctly. Being away for a week kind of took me away from the agreed upon processes so I need to get familiar with them.
- I am struggling with: The git workflow. Yesterday I also struggled but overcame some new-to-me git/github issues.
- Special thanks to: stackoverflow lol
- Something else random about the day: I would typically go train jiu jitsu tonight, but I'm taking the night off because I tweaked my neck on Tuesday (I'm fine). So instead I'm catching up with a few old friends at a coffee shop this evening.

**Comment by emilio12345** [comment]
1. What did you work on today? - I haven't worked on the project today but should have some time this afternoon or evening to take a look at @mnichols08 's feedback on my pull request from yesterday.

3. What are you struggling with? - I'm being challenged by the js portion of the project, but I'm already learning new things. 

4. Who deserves a shout out? - stackoverflow again and mdn docs :) 

5. Something else about your life non-project related to share. - I made it to 6am jiu jitsu class. I woke up early so decided to go.

**Comment by PaulaR-05** [comment]
1. **Today** I: I fixed some things in a function that I'm responsible for. Happy that it ended up working \o/

2. **I am struggling with:** Making time for studying in general :(

3. **Special Thanks to:** @mnichols08 @emilio12345 @Emimint & @Nazile-Tag , for the hard work you all have put into the project  this week :)

4. **Something else random about your day:** I didn't have time/energy yesterday to prepare dinner. So, tonight I'm gonna make Thai Cashew Chicken stir fry. https://www.recipetineats.com/thai-cashew-chicken-stir-fry/ . 🧑‍🍳

Hope you all have a lovely weekend and Happy Friday  🙌
![FridayPaydayGIF](https://user-images.githubusercontent.com/118011195/229205095-44a6fd17-abe7-450e-a7d3-4646dd9e12b5.gif)

**Comment by mnichols08** [comment]
Hello Everyone hope you are enjoying your weekends. I reviewed a few pull requests and I came home and did some pair programming. We built a stopwatch.

I am struggling with knowing if I am being overly protective of the codebase. I fear I am slowing production by requesting changes so often.

I want to say thanks to all of you for putting up with my change requests

I need to sleep now but I wanted to post this before I did sleep.

**Comment by PaulaR-05** [comment]
1. **Today** I: I took a class from a CSS Course :) 

2. **I am struggling with**: Making time for all things we have to do in life :(

3. **Special Thanks to:** my brother who did the grocery for me this week 😁

4. **Something else random about your day:** I got back to the gym today after the flu that I had o///

**Comment by mnichols08** [comment]
1. Today I decided to push some changes on our repository to get us set up with Webpack and SASS so we can speed up production. I spent a little time updating five stories in the format we should have it in to dynamically render the page for at least five stories. I actually managed to overcome a [coding problem](https://github.com/chingu-voyages/v43-tier1-team-09/discussions/82#discussioncomment-5471648) I was facing a few days ago. We can now dynamically render both input pages and story pages using the variables property on each scenario. [Check it out!](https://mnichols08-forked-madlibs.netlify.app/) | [Code](https://github.com/mnichols08/mad-libs/blob/main/src/views/renderStoryPage.js)

2. I am struggling with not being sleepy at all at night - but so exhausted beyond death in the morning that I need 8 cups of coffee to function and it is a never ending cycle.

3. Special Thanks to my mom for everything she does

4. Today it was almost like a tornado was going to happen with so much wind. On my way home from work I drive over a  mountain top and there was an updraft of leaves just lifting off the road and into the air up up and away it was a trip.

**Reply by PaulaR-05** [reply]
1. Thanks a lot for all the effort, Mikey! 🤗 
2. oh that's not good, sometimes it's good to slow down and rest :) 
3. moms are the best, always.
4. omg!

**Comment by emilio12345** [comment]
- What did you work on today? - Today I plan on doing some more work around the tasks assigned to me in Sprint 4. Though, one or both may need to roll over to Sprint 5.

- What are you struggling with? - Time. I had a busy work week and did not have much time to dedicate to this project. This week should be better.

- Who deserves a shout out? - Stackoverflow and MDN docs helped me a lot this past week :) 

- Something else about your life non-project related to share. - We are months late on getting our garden started. This weekend hopefully we'll have some time to get our small garden section ready to start planting.

**Reply by PaulaR-05** [reply]
Nice Mili, send pics from the garden when it gets done :)

**Comment by PaulaR-05** [comment]
1. Today I: I'm working in a style that allows users to print a madlib.
2. I am struggling with: Making time for all things we have to do in life :(
3. Special Thanks to: for the time spent with family on Sunday!
4. Something else random about your day:I might go again to the gym, get hopes up for it hahaha

**Comment by mnichols08** [comment]
1. Today I didnt really code much I came home and did some much needed cleaning
2. I am struggling with getting the right amount of sleep and also experiencing some eye fatigue staring at the screen all of the time
3. Special thanks to my boss for letting me work overtime this week. The extra money is nice and on top of that
4. Today I forgot to pack my lunch =( But the owner of the store that I work at gave me his credit card to get all of us fresh deli made sandwiches from a local deli [Schrock's County Store](https://www.schrockscountrystore.com/). It was so delicious, I really have the best boss. He deserves more shout-outs but here's mine <3

**Reply by PaulaR-05** [reply]
awn, I loved that place Schrock's County Store!
Also, yeap your boss is super nice! o/ ❤️❤️❤️

**Comment by emilio12345** [comment]
- What did you work on today? - Today I plan to pull this branch and test running the code locally - https://github.com/chingu-voyages/v43-tier1-team-09/tree/fix-96/folder-structure/mnichols08
- What are you struggling with? - Finding enough time for the project.
- Who deserves a shout out? - Team 09. Thank you all for continuing to work together. I know everyone is doing as much as they can.
- Something else about your life non-project related to share. - I sprained my ring finger in jiu jitsu last night. It doesn't affect my typing but the pain is distracting. I'll be fine though.

**Comment by PaulaR-05** [comment]
1. Today: write a bad word function + we checked if a web pack is working. 
2. I am struggling with this js part of the project is hard haha oh gosh!
3. Special thanks to: everyone, thanks for all your hard work.
4. Today I didn't go to the gym and order a Mcdonald's. shame on me

![IllusCuteGIF](https://user-images.githubusercontent.com/118011195/229963964-4fb86715-7736-46dc-b234-afdb967fe2ed.gif)

**Reply by mnichols08** [reply]
what a gif!

You deserve the Mickey Ds though for the dirty word function XD

**Reply by PaulaR-05** [reply]
I haven't read so many bad words in my whole life haha

**Comment by mnichols08** [comment]
Today Gabriela showed me her function she came up with and we were able to fill in a bunch of words from a github repository and get it working =) Just need to push some code to implement in our project

I am struggling to find time to do everything I want to do and 

Thanks to everyone for sticking in this, we will be making it look pretty soon enough

today I washed my car  and changed her air filter =)

**Comment by emilio12345** [comment]
- What did you work on today? - I had a quick call with Mikey and we tried to get node installed on my local machine but ran into some unknown issues. For now, I will try using github codespaces.
- What are you struggling with? - Having available time to continue work on the project and just errors/issues I get every time I try to do anything.
- Who deserves a shout out? - Mikey. Thank you!
- Something else about your life non-project related to share. - The weather here today is beautiful.

**Comment by PaulaR-05** [comment]
1. Today: upload a branch that has the web pack installed + I corrected a mistake on a print file I was working with.
2. I am struggling with having time. Today I got a problem with my PC and lost all the work I had done during the day. So, yeah I had to do it all again 😭 
3. Special thanks to: Mikey, for all the work he has done helping us. Thanks Mikey!
4. Today I went to the gym at 5 AM after eating Mcdonald's yesterday haha (balance is everything in life xD)

**Comment by mnichols08** [comment]
Wow I have neglected this a few days. Sorry about that.
Today: I haven't done much with the project, but I did update an issue title.
I've been struggling to keep up with this standup. 
Special Thanks to all of my coworkers today for putting up with me working on my car in between customer waves
I ended up having two cars at work today so I decided to go to my hometown for lunch and bum a ride back to work to get the other car home XD. Big thanks to my coworkers and boss - Even though it's not the dream job it's really a great place to work.

**Comment by mnichols08** [comment]
Today I have been working on implementing a dynamic header and footer. I was able to pull it off where the dots will migrate off screen / new ones appear. I really had a blast making this work. There is more we could do, I think it would be wise to push history into the browser so the user can use the back button as well. 
I struggled to figure out a way to not get the dots to re-render if you run the init function back to back so there is a mild bug to work on there if we push that code
Special Thanks to Emilie for coming through with the python script that sorted out our data today (right in the knick of time!)
We had a big family gathering today and when the little ones came around I was able to entertain them with our App, and it was just as @Emimint pushed a major update to the JSON that let them test out our app. It was so cool to see them laugh at some of the jokes. I noticed they were able to type peacock and it passed the filter. I wasn't anticipating for it to let it go through. I think you can sneak cuss words in if you surround them with other characters. But it was useful to see them use the App.

**Reply by PaulaR-05** [reply]
awnnnn, so nice! I wish I could see them playing!

**Reply by mnichols08** [reply]
It did wonders to visualize a better user experience!

**Comment by PaulaR-05** [comment]
1. Today: upload a branch with a readme update with the great work @Emimint Did with python + upload the meet our team page
2. I am struggling with having time. But I knew that It would be a problem... so, Let's do it o/
3. Special thanks to my parents for coming home and having lunch with me yesterday <3 
4. Something else about your life non-project related to share: I burned my hand cooking a lasagna. Silly me
